File: source/code/nodejs/quickstart/authentication_attempt.js
var usernamePasswordRequest = {
  username: 'tk421@stormpath.com',
  password: 'Changeme1'
};

application.authenticateAccount(usernamePasswordRequest, function (err, result) {
  if (err) {
    return console.error(err);
  }

  var account = result.account;

  console.log('Authenticated as %s with email %s', account.username, account.email);
});

File: source/code/java/quickstart/authentication_attempt.java
  String usernameOrEmail = "tk421@stormpath.com";
  String rawPassword = "Changeme1";

  // Create an authentication request using the credentials
  AuthenticationRequest request = UsernamePasswordRequest.builder()
      .setUsernameOrEmail(usernameOrEmail)
      .setPassword(rawPassword)
      .build();

  //Now let's authenticate the account with the application:
  try {
      AuthenticationResult result = application.authenticateAccount(request);
      account = result.getAccount();
      System.out.println(
          "Authenticated Account: " + account.getUsername() + ", " +
          "Email: " + account.getEmail()
      );
  } catch (ResourceException ex) {
      log.error(ex.getMessage());
  }
